Flow.Launcher/Converters/IconRadiusConverter.cs

```diff
@@ -8,15 +8,10 @@ public class IconRadiusConverter : IMultiValueConverter
     {
         public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            if (values.Length != 2)
-                throw new ArgumentException("IconRadiusConverter must have 2 parameters");
+            if (values is not [double size, bool half])
```

Comment:
The original code didn't have the variable names, I didn't know how to name them, so I just named them `size` and `half`.